server/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:10
+
+# If PORT variable is not passed as argument during image build, it defaults to 4000
+# i.e. docker build --build-arg PORT=${ANY PORT} ...
+ARG PORT=4000
+ENV PORT=${PORT}
```

Comment:
It probably doesn't make sense to treat this as a build arg, so you can just omit the `ARG` line here and set the default value directly on the `ENV` line